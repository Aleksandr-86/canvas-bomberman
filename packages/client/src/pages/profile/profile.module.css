.profile {
  display: flex;
  flex-direction: column;
  gap: 50px;
  justify-content: space-between;
  align-items: center;
  width: 900px;
  margin-top: 30px;
}

.profileContent {
  display: flex;
  flex-wrap: wrap;
  justify-content: space-evenly;
  width: 100%;
}

.profileAction {
  display: flex;
  flex-direction: inherit;
  align-items: center;
  width: 100%;
}

.actionButton {
  margin-bottom: 30px;
}

.buttonLink {
  padding: 10px 20px;
  border: none;
  background: none;
  color: blue;
  font-size: 40px;
  font-family: 'Rubik Bubbles', fantasy;
  text-decoration: none;
  text-transform: uppercase;
  cursor: pointer;
}

.buttonDanger {
  padding: 10px 20px;
  border: none;
  background: none;
  color: red;
  font-size: 40px;
  font-family: 'Rubik Bubbles', fantasy;
  text-decoration: none;
  text-transform: uppercase;
  cursor: pointer;
}
