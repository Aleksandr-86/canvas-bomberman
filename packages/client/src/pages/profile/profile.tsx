import styles from './profile.module.css'

export const Profile = () => {
  return <div className={styles.profile}>Страница профиля пользователя</div>
}
