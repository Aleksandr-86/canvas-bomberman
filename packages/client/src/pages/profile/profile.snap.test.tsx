import * as renderer from 'react-test-renderer'
import { Profile } from './profile'

describe(`Snapshot test page Profile`, () => {
  test(`Should correct render (without props)`, () => {
    const tree = renderer.create(<Profile />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
