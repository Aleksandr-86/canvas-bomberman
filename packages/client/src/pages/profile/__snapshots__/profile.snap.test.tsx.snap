// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test page Profile Should correct render (without props) 1`] = `
<div
  className="profile"
>
  Страница профиля пользователя
</div>
`;
